students/hyska_monika/lesson_06_dicts_tuples_sets_args_kwargs/functions_lesson06.py

```diff
@@ -76,3 +81,13 @@ def file_to_list(file_name):
     my_file = open(file_name, "r")
     words_list = my_file.read().split()
     return words_list
+
+
+# function change dictionary {'Poland' : ['Warsaw', 'Krakow']}
+# to dictionary  {'Warsaw': 'Poland, 'Krakow': 'Poland'}
+def change_dict_without_list(inventory):
```

Comment:
English: "change" is not very specific, it may mean almost everything. What you are making here could be called "index" or a "reverse mapping". For me to skip reading the content, I would have to see something like:
```
def city2country(country2cities):
def city2country_mapping(country2city_list):
def get_city2country_mapping(country2cities_mapping):
```